modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PageMemoryStorageEngine.java

```diff
@@ -109,4 +107,18 @@ public TableStorage createTable(TableConfiguration tableCfg) {
 
         return new VolatilePageMemoryTableStorage(tableCfg, dataRegion);
     }
+
+    /**
+     * Creates, starts and adds a new date region to the engine.
+     *
+     * @param name Data region name.
+     * @param dataRegionConfig Data region configuration.
+     */
+    private void addDataRegion(String name, PageMemoryDataRegionConfiguration dataRegionConfig, int pageSize) {
```

Comment:
Why do we need the `pageSize` parameter? Can it be computed based on the `engineConfig` field?

Reply:
Yes, I can fix it.